neo/Core/Blockchain.py

```diff
@@ -128,7 +128,7 @@ def GenesisBlock():
                      True)
 
     @staticmethod
-    def Default():
+    def Default() -> 'Blockchain':
```

Comment:
in this case yes, because we cannot type annotate a class that's not yet defined. Python's 3.7+ has a future feature I believe called forward annotations.

Reply:
yes because we cannot type annotate classes that are not yet defined. This is only possible in python 3.7+ with some future forward annotations feature.